### ISSUE TITLE
Mysql Error

### ISSUE THREAD

**Issue by msaad2110** [splewis/get5-web#154]
Hello there,
I have installed the web panel accoriding to your guide but still I'm facing this error when I try to open web page.
Sorry, unexpected error: Can't load plugin: sqlalchemy.dialects:ysql
Currently I'm using debian 8 OS.

**Comment by PhlexPlexico** [comment]
Are you using Python 2.7? Did all the requirements install successfully?

**Comment by msaad2110** [comment]
Yes I have installed all the things accoriding to your tutorial.

**Comment by PhlexPlexico** [comment]
Can you please post what is in the `get5.log` located under the logs directory? From the looks of it, there's no reference to `ysql` in this project. Is there by chance a typo that was made?

**Comment by PhlexPlexico** [comment]
Also, in your `prod_config` do you have the driver typed as `ysql://...` by chance?

**Comment by msaad2110** [comment]
that was just typo error now I'm receiving this error when I try to login through steam Sorry, unexpected error: argument of type 'int' is not iterable.

and getting this error in another vps which is ubuntu 16.
Sorry, unexpected error: (_mysql_exceptions.OperationalError) (2006, 'MySQL server has gone away') [SQL: u'SELECT user.id AS user_id, user.steam_id AS user_steam_id, user.name AS user_name, user.admin AS user_admin \nFROM user \nWHERE user.id = %s'] [parameters: (1,)]

**Comment by msaad2110** [comment]
and some times this whenever I restart mysql server after some time this error occurs in ubuntu 16.
(_mysql_exceptions.OperationalError) (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")

**Comment by PhlexPlexico** [comment]
See #146 if using MariaDB, that should fix those problems.

**Comment by msaad2110** [comment]
I'm facing same issue still just upgraded the server too MariaDB but yet facing this error while I try to login it from steam..
 Sorry, unexpected error: argument of type 'int' is not iterable

**Comment by PhlexPlexico** [comment]
And what does the error log in get5 say (`get5.log` located in the logs directory)? Where is the error coming from?

**Comment by msaad2110** [comment]
here it is 

ile "/var/www/get5-web/get5/__init__.py", line 122, in create_or_login
    g.user = User.get_or_create(steam_id)
  File "/var/www/get5-web/get5/models.py", line 33, in get_or_create
    steam_id in app.config['ADMIN_IDS'])
TypeError: argument of type 'int' is not iterable
[2019-05-23 20:16:02,080] {/var/www/get5-web/get5/__init__.py:155} ERROR - argument of type 'int' is not iterable

**Comment by PhlexPlexico** [comment]
It's your Admin_IDS. They cannot be int, but they need to be strings. I assume you have it set as:
`ADMIN_IDS = [1234567890]` Where 12345676890 is your steam ID. Please set it as a string as such.
`ADMIN_IDS = ['1234567890']`.

**Comment by msaad2110** [comment]
Did it,
How to add multiple ID's in it?
ADMIN_IDS = ['1234567890','81612919']
like this?

**Comment by TandelK** [comment]
> Did it,
> How to add multiple ID's in it?
> ADMIN_IDS = ['1234567890','81612919']
> like this?

Yes @msaad2110 thats the way you have to use Admins IDs

**Comment by msaad2110** [comment]
I'm admin only I can create public team/servers can't add players xyz or control matches created by other users?

**Comment by msaad2110** [comment]
I have added the admin id in the list but still admin can only public team / server.
they are not able to manage other matches like they can't see admin options

**Comment by TandelK** [comment]
Did you enable option in prod_config.py of this line
https://github.com/splewis/get5-web/blob/259472f035c3096d998890a7f8d79ee04dccde4d/instance/prod_config.py.default#L21 
You have to set that function to True and saving do a apache service restart command.
